packages/client/components/BackButton.tsx

```diff
@@ -17,6 +17,8 @@ const IconButton = styled(FlatButton)({
 
 const BackIcon = styled(ArrowBack)({
   color: 'inherit'
+  position: 'fixed',
+  bottom: '94%'
```

Comment:
This doesn't look good, why hardcode something that looks like a very arbitrary number?